tasks/arclight.rake

```diff
@@ -20,6 +20,7 @@ end
 
 desc 'Run Eslint'
 task :eslint do
+  system 'yarn install'
```

Comment:
It's unfortunate to have to run this on every build (it's relatively fast, but still).  I could imagine doing this on travis in a `before_script` or something like that, but then that would leave the potential for a developer to not have installed the dependencies.  We could put at higher up as part of our install process, which would alleviate this need, but wasn't sure where we would/should slot that in.

Reply:
We could do it on travis in a before_script and then alter the task to fail if eslint is not installed, perhaps?

Reply:
Hmm, interesting idea.  Let me poke at that.